### ISSUE TITLE
Not working again..please update!

### ISSUE THREAD

**Issue by DjPari** [sucresware/voicemeeter-activator-autoit#7]
it worked as recent as probably a month ago from this post....booted up my pc today..got the popup and proceeded to apply the patcher and id did not work,,,i even rebooted to make sure

**Comment by fluffeliger** [comment]
+1 not working

**Comment by ghost** [comment]
Guys it's not working because of the newer version, if you want it to work unistall your current build of Voicemeeter and use this one instead (I use VoiceMeeter Bannana, so this is the only link that I personally found) [https://download.vb-audio.com/Download_CABLE/Voicemeeter8Setup_v3013.zip](url)

**Comment by ghost** [comment]
Btw if you click it, it won't work, so you will have to copy and open it in a new tab.

**Comment by ghost** [comment]
(If you don't trust me this fix wasn't found by me it was found in the comments of the first issue) [https://github.com/sucresware/voicemeeter-activator-autoit/issues/1#issuecomment-1279826288](url)

**Comment by soonos** [comment]
+1 not working

**Comment by ghost** [comment]
> +1 not working

Yeah sorry, they took it down, but luckily it still works with the waybackmachine [](https://download.vb-audio.com/Download_CABLE/Voicemeeter8Setup_v3013.zip) (You might have to copy and paste the link into a new tab because sometimes it doesn't work)

**Comment by Grenade-Inspector** [comment]
https://download.vb-audio.com/Download_CABLE/Voicemeeter8Setup_v3013.zip

**Comment by ghost** [comment]
https://web.archive.org/web/20200430081803/https://download.vb-audio.com/Download_CABLE/Voicemeeter8Setup_v3013.zip

**Comment by jeyan050** [comment]
> > +1 not working
> 
> Yeah sorry, they took it down, but luckily it still works with the waybackmachine (You might have to copy and paste the link into a new tab because sometimes it doesn't work)

Ive installed the older version using ur link, but when I still open the VMactivator, it doesnt work

**Comment by ghost** [comment]
> 

hmm usually it works for me, so I would either try to run the activator again, but in admin mode or go to control panel, then find the voicemeeter program and uninstall it, then restart your system and reinstall the older voicemeeter version, then run the activator in admin mode.

(There's a chance that they have tampered with that web archive link because github usually does the same for malicious software, so I'll include a link to the installer that I use.)

https://mega.nz/file/PxAFgBpL#mYjxwAPjOrFpwhNCMGKVTyyy7Kki9eUvUzv8-7N4mGY Mega might ask for a decryption key, so I'll include that here as well. 

mYjxwAPjOrFpwhNCMGKVTyyy7Kki9eUvUzv8-7N4mGY <--- Decryption Key

**Comment by jeyan050** [comment]
> > 
> 
> hmm usually it works for me, so I would either try to run the activator again, but in admin mode or go to control panel, then find the voicemeeter program and uninstall it, then restart your system and reinstall the older voicemeeter version, then run the activator in admin mode.
> 
> (There's a chance that they have tampered with that web archive link because github usually does the same for malicious software, so I'll include a link to the installer that I use.)
> 
> https://mega.nz/file/PxAFgBpL#mYjxwAPjOrFpwhNCMGKVTyyy7Kki9eUvUzv8-7N4mGY Mega might ask for a decryption key, so I'll include that here as well.
> 
> mYjxwAPjOrFpwhNCMGKVTyyy7Kki9eUvUzv8-7N4mGY <--- Decryption Key

Oh kk, will do, but before I do, I might have been stupied and doing it wrong. Am I supposed to open and run the older version of Voicemeter and then the activator? or is it only the activator (this is the one Ive been doing)?

**Comment by ghost** [comment]
> 

Sorry, but I don't fully understand what you're asking, but I'm assuming that your question is about whether to run the activator with a newer or older version of voicemeeter. You want to have the older version of voicemeeter open and running, then open the activator and click the activate button.

All the program really does is edit a registry key, I'm pretty sure, so whether voicemeeter is open or not doesn't really matter.

**Comment by jeyan050** [comment]
> > 
> 
> Sorry, but I don't fully understand what you're asking, but I'm assuming that your question is about whether to run the activator with a newer or older version of voicemeeter. You want to have the older version of voicemeeter open and running, then open the activator and click the activate button.
> 
> All the program really does is edit a registry key, I'm pretty sure, so whether voicemeeter is open or not doesn't really matter.

Sorry, I might have confused you.
I wasn't sure if I was running the activator correctly. I was doing before that I would close VoiceMeeter Potato, and then run the Activator itself. However, as you stated, I'm supposed to run and leave VoiceMeeter Potato open AND then run the activator.

Now, back to running the older version of VM potato. After I uninstalled the latest version and installed the 3.0.1.3 version, there's no popup of the donationware anymore, even without running the activator. Most likely because its redoing the trial w/ no popups of donations. I'll most likely try it again when the trial runs out

**Comment by ghost** [comment]
> > > 
> > 
> > 
> > Sorry, but I don't fully understand what you're asking, but I'm assuming that your question is about whether to run the activator with a newer or older version of voicemeeter. You want to have the older version of voicemeeter open and running, then open the activator and click the activate button.
> > All the program really does is edit a registry key, I'm pretty sure, so whether voicemeeter is open or not doesn't really matter.
> 
> Sorry, I might have confused you. I wasn't sure if I was running the activator correctly. I was doing before that I would close VoiceMeeter Potato, and then run the Activator itself. However, as you stated, I'm supposed to run and leave VoiceMeeter Potato open AND then run the activator.
> 
> Now, back to running the older version of VM potato. After I uninstalled the latest version and installed the 3.0.1.3 version, there's no popup of the donationware anymore, even without running the activator. Most likely because its redoing the trial w/ no popups of donations. I'll most likely try it again when the trial runs out

Alright I believe the trial period is around 90 days.

**Comment by Garze23** [comment]
Hello, what is the difference between the 2 versions (3.0.2.8 and 3.0.1.3)?

**Comment by ghost** [comment]
> Hello, what is the difference between the 2 versions (3.0.2.8 and 3.0.1.3)?

Sorry for the REALLY late response, but the difference is simply age, meaning that one blocks this activator and the older one doesn't, so that's why the one I specifically linked to works, while the newer releases of Voicemeeter doesn't, I think most of the features are the same, but I've never looked into it.
